packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -61,13 +61,16 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
 
   const [searchMember, setSearchMember] = useState("");
   const debouncedSearchMember = useDebounce(searchMember, 500);
-  const oooForMembers = trpc.viewer.teams.legacyListMembers.useInfiniteQuery(
-    { limit: 10, searchText: debouncedSearchMember },
-    {
-      enabled: true,
-      getNextPageParam: (lastPage) => lastPage.nextCursor,
-    }
-  );
+  const oooForMembers =
+    oooType === OutOfOfficeTab.TEAM
+      ? trpc.viewer.teams.legacyListMembers.useInfiniteQuery(
+          { limit: 10, searchText: debouncedSearchMember, adminOrOwnedTeamsOnly: true },
+          {
+            enabled: true,
+            getNextPageParam: (lastPage) => lastPage.nextCursor,
+          }
+        )
+      : undefined;
```

Comment:
We should avoid conditional hooks. Let's just use the condition to enable the query.

```suggestion
  const oooForMembers = trpc.viewer.teams.legacyListMembers.useInfiniteQuery(
    { limit: 10, searchText: debouncedSearchMember },
    {
      enabled: oooType === OutOfOfficeTab.TEAM,
      getNextPageParam: (lastPage) => lastPage.nextCursor,
    }
  );
```